siphon/tests/test_acis.py

```diff
@@ -0,0 +1,54 @@
+"""Test ACIS Web Services API Access."""
+from siphon.simplewebservice.acis import acis_request
+from siphon.testing import get_recorder
+
+recorder = get_recorder(__file__)
+
+
+@recorder.use_cassette('acis_request')
```

Comment:
I used one "cassette" for all 5 tests.. Is that alright? They're all JSON strings.

Reply:
I think that's fine for now. If we ever need to regenerate, we'll have to separate them, since the cassette is only ever generated from a single test.